sdk/js/src/Client.ts

```diff
@@ -239,6 +241,10 @@ export class DVCClient implements Client {
         await this.eventQueue.close()
     }
 
+    get closing(): boolean {
```

Comment:
is this still needed?

Reply:
I figured its useful to have regardless in case you need to tell externally that close has been called